active/0000-nested-addon-imports.md

```diff
@@ -0,0 +1,63 @@
+- Start Date: (fill me in with today's date, 2016-08-06)
+- RFC PR: 
+- ember-cli Issue: 
+
+# Summary
+
+The new asset import API, available since ember-cli 2.7.0, offers the possibility to 
+import Ember CLI addons from Ember CLI addons (aka, nested addons).
+This RFC intends to find solutions for typical edge cases.
+
+# Motivation
+
+Since ember-cli 2.7.0, addons can be nested if the nesting floows the following
+scenario:
+* The nested addons do not make use of the property `parent.bowerProperty` to define
+  assets paths because it not always defined (it is not defined in EmberAddon instances)
+* The nested addons do not need to execute a default blueprint.
+
+Solving these use cases would allow for a simpler, more generic, addons development
+process, and will allow teams to build more reusable and higher level addons for
+their projects and the community (like [ember-admin](https://github.com/DockYard/ember-admin)
+using [ember-power-select](ember-power-select.com) with no additional boilerplate, for instance).
+
+# Detailed design
+
+Here is a rundown of the questions raised with @nathanhammond in [a previous PR](https://github.com/ember-cli/ember-cli/pull/6111):
+
+1. Should nested addons be allowed to use their grandparents' `bower_components` directory?
+1. How/When should the default blueprints of the nested be run?
+1. How to deal with conflicting versions (The app uses my-addon 1.0.0, but my-addon 2.0.0 is required as a nested component)?
+
+For each of these questions, here's a first attempt towards an answer:
+
+1. Yes, they should, because there's only one bower directory available to the app, and it should be available to the whole app,
+  including to the dependencies. See [the PR linked above](https://github.com/ember-cli/ember-cli/pull/6111) for an example.
```

Comment:
My concern with your proposal is that by implementing it, we will build a better/safer Bower in ember-cli on top of Bower, which, as you made clear, we want to move away from.

Looking towards what's "best", I agree with your description, but IMHO, it is an order of magnitude more complicated than warning the user that "though Bower is supported, it appears that installing AddonX requires the installation of dependencyA#1.0.0 which conflicts with the currently installed dependencyA#2.0.0".

A message does not solve anything but at the same time it does not cost much, and maybe that's enough for  a part of the stack that will eventually be deprecated.